src/attr.rs

```diff
@@ -34,6 +34,18 @@ pub fn get_attrs_from_stmt(stmt: &ast::Stmt) -> &[ast::Attribute] {
     }
 }
 
+pub fn get_span_without_attrs(stmt: &ast::Stmt) -> Span {
+    match stmt.node {
```

Comment:
If I get it right, the test only hits `ast::StmtKind::Mac`. Could you add more statements to the test so that the rest of `ast::StmtKind` values, e.g., `Local`, are also tested.

Reply:
Done!